.github/cmake/ctest_test.cmake

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.8)
+
+include("${CMAKE_CURRENT_LIST_DIR}/github_ci.cmake")
+
+# Read the files from the build directory.
+ctest_read_custom_files("${CTEST_BINARY_DIRECTORY}")
+# Uncomment the line below and all ctest_submit_multi lines
+# if CTestConfig.cmake defines `drop_sites`.
+# include("${CMAKE_CURRENT_LIST_DIR}/ctest_submit_multi.cmake")
+
+# Pick up from where the configure left off.
+ctest_start(APPEND)
+
+include(ProcessorCount)
+ProcessorCount(nproc)
+
+include("${CMAKE_CURRENT_LIST_DIR}/ctest_exclusions.cmake")
+ctest_test(
+  PARALLEL_LEVEL "${nproc}"
+  RETURN_VALUE test_result
+  EXCLUDE "${test_exclusions}")
+
+file(GLOB packages
+  "${CTEST_BINARY_DIRECTORY}/PluginTests/*/build/plugin/build/*.tar.gz")
```

Comment:
Do we actually generate these files?

Reply:
I've commented this out but would like to leave it there should we decide to upload binaries to the dashboard.